src/main/java/org/walkmod/javalang/compiler/types/CachedClassLoader.java

```diff
@@ -14,35 +14,52 @@
  */
 package org.walkmod.javalang.compiler.types;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
Style: wild or no wild?

Reply:
It has been automatically performed by Intellij. I will removed it.